src/ds5/ds5-device.cpp

```diff
@@ -839,11 +870,11 @@ namespace librealsense
 
     double ds5_device::get_device_time_ms()
     {
-        // TODO: Refactor the following query with an extension.
-        if (dynamic_cast<const platform::playback_backend*>(&(get_context()->get_backend())) != nullptr)
-        {
-            throw not_implemented_exception("device time not supported for backend.");
-        }
+        //// TODO: Refactor the following query with an extension.
+        //if (dynamic_cast<const platform::playback_backend*>(&(get_context()->get_backend())) != nullptr)
```

Comment:
Why?

Reply:
The global timestamp thread sends hw monitor timestamp query, which gets recorded.
If we disable this query by the playback, the playback will get stuck / mismatch or other undesired behavior.